src/Symfony/Bundle/FrameworkBundle/Resources/config/web.xml

```diff
@@ -67,6 +67,12 @@
             <argument type="service" id="router" on-invalid="ignore" />
         </service>
 
+        <service id="http_exception_listener" class="Symfony\Component\HttpKernel\EventListener\HttpExceptionListener">
```

Comment:
Actually currently the build-in simple logger service is registered during a compiler pass completely:

https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpKernel/DependencyInjection/LoggerPass.php#L38

So wondering if it should be the same here? Add service definition inside compiler pass if no templating? That's a question for one of the core members I guess.